### PR TITLE
BaSP-MR-87: admins GET method on redux

### PR DESCRIPTION
**Pull request by Bigzufi**
- Get method, with pending, success and error
- Fetch request added in Thunks
- Combine reducers
- Adding loader to component

### CODE REVIEW

File: src/redux/admins/reducer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { combineReducers } from 'redux';
+import { GET_ADMINS_PENDING, GET_ADMINS_SUCCESS, GET_ADMINS_ERROR } from './constants';
```
Comment (DelfiLT) [inline src/redux/admins/reducer.js:2]: For a recomendation you could use `impor * as name from` to not call one by one

File: src/redux/admins/actions.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { GET_ADMINS_PENDING, GET_ADMINS_SUCCESS, GET_ADMINS_ERROR } from './constants';
```
Comment (DelfiLT) [inline src/redux/admins/actions.js:1]: For a recomendation you could use impor * as name from to not call one by one

File: src/redux/admins/thunks.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { getAdminsPending, getAdminsSuccess, getAdminsError } from './actions';
```
Comment (DelfiLT) [inline src/redux/admins/thunks.js:1]: For a recomendation you could use impor * as name from to not call one by one

**Review by DelfiLT — APPROVED** [review]

**Review by SantiH97 — APPROVED** [review]
Nice work :)

**Pull request merged**